docs/types/reference-types.rst

```diff
@@ -376,20 +376,20 @@ Array Members
     pragma solidity >=0.6.0 <0.9.0;
 
     contract ArrayContract {
-        uint[2**20] m_aLotOfIntegers;
+        uint[2**20] mLotsOfIntegers;
```

Comment:
This is not snake case naming. It rather looks like a prefix to indicate it's a member variable, which is a separate convention. Whether these prefixes should be kept or not is another question but I think that just mangling them by removing the underscore is not a good idea because it makes this convention unrecognizable.